core/src/main/java/hudson/model/Queue.java

```diff
@@ -1,9 +1,12 @@
 /*
  * The MIT License
- *
+ * 
+ * Copyright (c) 2018-2019 Intel Corporation
+ * Copyright (c) 2015-2017, Intel Deutschland GmbH
```

Comment:
Just add "and other contributors to the original title?"

Reply:
Due to legal requirements, Intel policy does not permit us to join the previous business entities under which Intel operated here in German.

This leads us to having to include the following copyrights depending on the year it was issued:

Copyright (c) 2018-2019 Intel Corporation
Copyright (c) 2015-2017, Intel Deutschland GmbH
Copyright (c) 2011-2014, Intel Mobile Communications GmbH

And yes, the lack of a comma in the 2018/2019 version is intentional. Isn't copyright law fun? :)

Reply:
As an ex-Intel employee, I know that feeling

What if...

* The original copyright is updated to `2004-2019` and to reference "all contributors" after it
* Intel copyrights are listed below the generic ones

Reply:
Hi Oleg.
I got in touch with our open source copyright legal contact and they got us a waiver that we can condense the copyrights, since we didn't actually release anything before our business entity had already been completely absorbed.

I will change this pull request and 4138 (that I uploaded at the same time) accordingly.

Reply:
thanks!